frontend/src/Pages/TrimSelectionPage/Trim/ChangeModal/UnlockItem/index.tsx

```diff
@@ -6,7 +6,7 @@ interface UnlockItemProps {
 function UnlockItem({ image, name }: UnlockItemProps) {
   return (
     <div className='w-full flex gap-3'>
-      <img src={image} alt={name} />
+      <img src={`https:\\${image}`} alt={name} className='w-[60px] h-[60px]' />
```

Comment:
```suggestion
      <img src={`https://${image}`} alt={name} className='w-[60px] h-[60px]' />
```

이미지 주소 이렇게 수정해주셔야 할 거예요.
저번에 색상 부분에서 이미지 안 뜨던 문제가 이것 때문이었어요.
이 부분 수정되면 머지할게요~

Reply:
어맛 내 알겠습니다!

Reply:
수정 확인했습니다!